Mage.Sets/src/mage/cards/b/BakisCurse.java

```diff
@@ -0,0 +1,101 @@
+ *  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ *  ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ *  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+ *  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *  The views and conclusions contained in the software and documentation are those of the
+ *  authors and should not be interpreted as representing official policies, either expressed
+ *  or implied, of BetaSteward_at_googlemail.com.
+ */
+package mage.cards.b;
+
+import java.util.UUID;
+import mage.abilities.Ability;
+import mage.abilities.effects.OneShotEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.filter.FilterPermanent;
+import mage.filter.predicate.mageobject.CardTypePredicate;
+import mage.filter.predicate.mageobject.SubtypePredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+
+/**
+ *
+ * @author L_J
+ */
+public class BakisCurse extends CardImpl {
+
+    public BakisCurse(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId,setInfo,new CardType[]{CardType.SORCERY},"{2}{U}{U}");
+
+        // Baki's Curse deals 2 damage to each creature for each Aura attached to that creature.
+        this.getSpellAbility().addEffect(new BakisCurseEffect());
+    }
+
+    public BakisCurse(final BakisCurse card) {
+        super(card);
+    }
+
+    @Override
+    public BakisCurse copy() {
+        return new BakisCurse(this);
+    }
+
+    private static class BakisCurseEffect extends OneShotEffect {
+
+    public BakisCurseEffect() {
+            super(Outcome.Detriment);
+            staticText = "Baki's Curse deals 2 damage to each creature for each Aura attached to that creature.";
+        }
+
+        public BakisCurseEffect(final BakisCurseEffect effect) {
+            super(effect);
+        }
+
+        @Override
+        public boolean apply(Game game, Ability source) {
+
+            FilterPermanent filterEnchantments = new FilterPermanent();
+            filterEnchantments.add(new CardTypePredicate(CardType.ENCHANTMENT));
+            filterEnchantments.add(new SubtypePredicate(SubType.AURA));
+            for (Permanent auraEnchantment : game.getBattlefield().getActivePermanents(filterEnchantments, source.getControllerId(), source.getSourceId(), game)) {
+                if (auraEnchantment.getAttachedTo() != null) {
+                    Permanent attachedToCreature = game.getPermanent(auraEnchantment.getAttachedTo());
+                    if (attachedToCreature != null && attachedToCreature.isCreature()) {
+                        attachedToCreature.damage(2, source.getId(), game, false, true);
+                        game.informPlayers("2 damage assigned to " + attachedToCreature.getName() + " from Baki's Curse due to " + auraEnchantment.getName());
```

Comment:
The damage should be simultaneous (i.e. count the number of auras, then double it and deal that much). Also a message is probably not necessary but if there is one it should reflect the total damage rather than the two at a time.